src/zenml/model/model.py

```diff
@@ -722,7 +723,18 @@ def _get_or_create_model_version(
                     retries_made += 1
             self.version = model_version.name
             self.was_created_in_this_run = True
+            model_version_url = dashboard_utils.get_model_version_url(
+                model_version.id
+            )
             logger.info(f"New model version `{self.version}` was created.")
+            if model_version_url:
+                logger.info(
+                    f"Model version dashboard URL: {model_version_url}"
+                )
+            else:
+                logger.info(
+                    "You can now view your Models in the ZenML dashboard using ZenML Cloud."
+                )
```

Comment:
```suggestion
            
            logger.info(f"New model version `{self.version}` was created.")
        model_version_url = dashboard_utils.get_model_version_url(
            model_version.id
        )
        if model_version_url:
            logger.info(
                f"Model version dashboard URL: {model_version_url}"
            )
        else:
            logger.info(
                "You can now view your Models in the ZenML dashboard using ZenML Cloud."
            )
```
This will make it visible for both cases: the new and existing model versions.